src/game.cpp

```diff
@@ -3363,10 +3363,10 @@ void Game::processClientEvents(CameraOrientation *cam, float *damage_flash)
 			//u16 damage = event.player_damage.amount;
 			//infostream<<"Player damage: "<<damage<<std::endl;
 
-			*damage_flash += 100.0;
-			*damage_flash += 8.0 * event.player_damage.amount;
+			*damage_flash += 63.0;
+			*damage_flash += 6.4 * event.player_damage.amount;
```

Comment:
That's not what I meant.
The damage flash increases each time the player gets hurt, so a maximal value of this variable should be added too, to prevent endless long red screens.